rowboat/models/message.py

```diff
@@ -197,7 +199,10 @@ def create_from_message_ids(cls, message_ids):
 
     @property
     def url(self):
-        return 'https://rowboat.party/archive/{}.txt'.format(self.archive_id)
+	with open('config.yaml', 'r') as f:
```

Comment:
spacing is fucked here, also ideally this should be extracted into a common area, but I can do that post-pr